src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/PosFutureDriftRuleTest.cs

```diff
@@ -21,7 +21,7 @@ public PosFutureDriftRuleTest()
         [Fact]
         public async Task RunAsync_HeaderTimestampTooNew_WithoutReducedDrift_ThrowsBlockTimestampTooFarConsensusErrorAsync()
         {
-            var futureDriftTimestamp = (PosConsensusValidator.DriftingBugFixTimestamp - 100);
+            var futureDriftTimestamp = (PosCoinviewRule.DriftingBugFixTimestamp - 100);
```

Comment:
Does `DriftingBugFixTimestamp` now below somewhere else rather than in the coinview rule? Any suggestions?

Reply:
Yes, in the `PosFutureDriftRule`